README.md

```diff
@@ -27,6 +27,23 @@ centrally managing routing policy and fronting service discovery.
 * [linkerd](linkerd/README.md)
 * [namerd](namerd/README.md)
 
+## Quick start ##
```

Comment:
Fair, I've added links from here to those other two README's. I'd still like to keep these commands front and center, as it's the first thing folks see when they hit the repo.